### DISCUSSION TITLE
VSCode and WSL: "_omp_hook: command not found"

### DISCUSSION THREAD

**Discussion by MMitsialis** [JanDeDobbeleer/oh-my-posh#3261] [Troubleshoot]
Hi,

I am running the application versions below, all seems to be working in standard and Microsoft Terminals. However I get the following error in the VSCode Remote WSL, and sometimes in Powershell Integrated Terminal in VSCode.  

Any pointers or suggested configuration changes would be appreciated? 

I can post additional config files if needed, I don't know what to offer at present beyond the below. 

```  shell
Windows 10 22H2 (patched to December 2022)
WSL (wit Ubuntu 22.04.1 LTS)
VSCode 
Version: 1.73.1 (user setup)
Commit: 6261075646f055b99068d3688932416f2346dd3b
Date: 2022-11-09T04:27:29.066Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19045
Sandboxed: No
```

## Error in VSCode Integrated terminal
![image](https://user-images.githubusercontent.com/10279937/209830111-7f03f9b5-0e92-48eb-978c-e0d13680e120.png)

**Comment by JanDeDobbeleer** [comment]
@MMitsialis I would remove the shell integrations function from vscode and see if the issue still occurs
